layouts/case-studies/single-baseof.html

```diff
@@ -6,7 +6,7 @@
         <body{{ with .Page.Params.body_class }} class="{{ . }}"{{ end }}>
 		{{ partial "navbar.html" . }}
 		{{ block "deprecation_warning.html" . }}
-		{{ partial "deprecation-warning.html" . }}
+		{{ partial "version-banner.html" . }}
```

Comment:
This is replacing a `<section>` with a `pageinfo`; I would wrap it or use a custom partial.

Out of the two options I prefer / recommend a custom partial (both `version-banner` and the custom partial can call common code, to help reuse).

Reply:
Is using a section necessary? Even the pageinfo shortcode that we are [using](https://kubernetes.io/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/) doesn't come with a `<section>`. I guess it would be more semantically correct to use a section but I wonder why Docsy doesn't use it.

> Out of the two options I prefer / recommend a custom partial (both version-banner and the custom partial can call common code, to help reuse).

I am sorry, but I am not sure what you mean here. If we have to use a `<section>` instead of a `<div>`, I would think that replacing the [`<div class="pageinfo pageinfo-{{ $color }}">`](https://github.com/kubernetes/website/pull/50217/files#diff-526e6786960d17aa6bcbb2dafc1713f395f78b8c2a8d241b17aa80085b59c48aR7) with `<section class="pageinfo pageinfo-{{ $color }}">` would be enough. How would a custom partial fit?

Reply:
AIUI the CSS for that page expects to have a set of `<section>` elements laid out using flexbox. And even it today it's not like that, one day someone might want to rely on the page consisting of sections.

For example:
```suggestion
+             {{ if .Site.Params.deprecated }}<section class="version-banner">{{ end }}
                {{ partial "version-banner.html" . }}
+             {{ if .Site.Params.deprecated }}</section>{{ end }}
```

Reply:
> AIUI the CSS for that page

Just to clarify, are you talking about the case study page in particular or all pages using the banner in general.

Reply:
This is specifically about the case study layouts. Other pages can rely on Docsy's default behavior.

Reply:
Ah, then it makes sense. I will rework the case study page then.

Alternatively, what do you say about reworking the case study page? It looks pretty broken in live but looks pretty ok in version 1.31 of the site. I was thinking of fixing it first.

Reply:
I say https://github.com/kubernetes/website/discussions/48432

Reply:
Nice, I will give it a shot.

Reply:
I am thinking that until we have a decision for the case studies page, I won't spend effort on it. I will update the `case-studies/single-baseof.html` and move on to other alignments.

Reply:
Well, with the closure of #50282 this should be resolved right?